physical/mysql/mysql.go

```diff
@@ -100,16 +100,39 @@ func NewMySQLBackend(conf map[string]string, logger log.Logger) (physical.Backen
 
 	db.SetMaxOpenConns(maxParInt)
 
+	// Check schema exists
+	var schemaExist bool
+	schemaRows, err := db.Query("SELECT SCHEMA_NAME FROM information_schema.SCHEMATA WHERE SCHEMA_NAME = ?", database)
+	defer schemaRows.Close()
+	if err != nil {
+		return nil, fmt.Errorf("failed to check mysql schema exist: %v", err)
+	}
+	schemaExist = schemaRows.Next()
+
+	// Check table exists
+	var tableExist bool
+	tableRows, err := db.Query("SELECT TABLE_NAME FROM information_schema.TABLES WHERE TABLE_NAME = ?", table)
```

Comment:
Shouldn't this be `dbTable`? I'm not well versed in mysql but it seems like this could match any table with the correct name from any database since it's not being scoped.